src/System.Private.CoreLib/shared/System/Collections/Generic/Dictionary.cs

```diff
@@ -426,8 +427,8 @@ public virtual void GetObjectData(SerializationInfo info, StreamingContext conte
                 }
             }
 
-            isFound = false;
-            return ref RuntimeHelpers.CreateNullRef<TValue>();
+            isFound = RuntimeHelpers.IsNullRef(in value);
```

Comment:
Is the `isFound` parameter even needed now? Seems like success can be determined solely by whether the function returns a null reference or a non-null reference.

Reply:
Removed